resources/docs/inertia/creating-chirps.md

```diff
@@ -99,16 +99,16 @@ Let's test our route and controller by returning a test message from the `index`
 namespace App\Http\Controllers;
 
 use App\Models\Chirp;
-use Illuminate\Http\Request;
-use Illuminate\Http\Response;
+use Illuminate\Http\Request; // [tl! collapse:end]
+use Illuminate\Http\Response; // [tl! 1 add]
 
-// [tl! collapse:end]
 class ChirpController extends Controller
 {
     /**
      * Display a listing of the resource.
      */
-    public function index(): Response
+    public function index()
+    public function index(): Response // [tl! remove:-1,1 add]
```

Comment:
Double method here.

Reply:
The comment will remove the previous line before adding a new one.